packages/docusaurus-theme-classic/src/index.js

```diff
@@ -109,3 +110,71 @@ module.exports = function (context, options) {
     },
   };
 };
+
+const NavbarLinkSchema = Joi.object({
+  items: Joi.array().optional().items(Joi.link('#navbarLinkSchema')),
+  to: Joi.string(),
+  href: Joi.string().uri(),
+  prependBaseUrlToHref: Joi.bool().default(true),
+  label: Joi.string(),
+  position: Joi.string().equal('left', 'right').default('left'),
+  activeBasePath: Joi.string(),
+  activeBaseRegex: Joi.string(),
+  className: Joi.string(),
+  'aria-label': Joi.string(),
+})
+  .xor('href', 'to')
```

Comment:
that's a cool feature, not in yup afaik :)

Reply:
it will help a lot of people in footer config.